MyScript/index.js

```diff
@@ -0,0 +1,67 @@
+//const inputDate = '11/27/2021'
+const inputDate = prompt('Enter the date You are interested in: ', '11/26/2021')
+const inputDateFull = new Date(inputDate)
+
+const checkEnteredDay = (enteredDate) => {
+    const twoDayMs = 86400000 * 2
+    const firstWorkDayMs = new Date(2021, 10, 26).getTime()
+
+    const inputDateMs = new Date(enteredDate).getTime()
+    const differenceMs = inputDateMs - firstWorkDayMs
+    const divisionResult = differenceMs / twoDayMs
+    const reminder = divisionResult % twoDayMs
+    const checkSecondDay = Math.round(reminder) % 2
+    const isFirstWorkDay = !(reminder % 2)
+    const isFirstFreeDay = !isFirstWorkDay && Number.isInteger(reminder)
```

Comment:
I think it's redundant to have so many variables and checks. For example, if ```isFirstWorkDay === true```, then it's automatically cannot be second one, we don't need second check. If it's false, then it means that day is the second one. Same goes for ```isFreeDay```